scripts/initializers/index.js

```diff
@@ -54,6 +54,7 @@ export default async function initializeDropins() {
 
     // Initialize Global Drop-ins
     await import('./auth.js');
+    await import('./company-switcher.js');
```

Comment:
This is a unified B2C/B2B Boilerplate. Initializing globally in all storefronts would break B2C storefronts. We need a solution that applies only to B2B storefronts.

A conditional load based on configuration? @sirugh, thoughts?

Reply:
> This is a unified B2C/B2B Boilerplate. Initializing globally in all storefronts would break B2C storefronts. We need a solution that applies only to B2B storefronts.

True, however at this point I couldnt' find a better way for it to work. We'll adjust it as soon as there is a commonly accepted way of doing it